deploy/rbac.yaml

```diff
@@ -6,31 +6,36 @@ rules:
 - apiGroups:
   - psmdb.percona.com
   resources:
-  - "*"
+  - perconaservermongodbs
   verbs:
-  - "*"
+  - "list"
+  - "update"
+  - "watch"
 - apiGroups:
   - ""
   resources:
   - pods
   - pods/exec
   - services
-  - endpoints
   - persistentvolumeclaims
-  - events
-  - configmaps
   - secrets
   verbs:
-  - "*"
+  - "create"
+  - "update"
+  - "delete"
+  - "get"
+  - "list"
 - apiGroups:
   - apps
   resources:
   - deployments
-  - daemonsets
```

Comment:
I think that we will need daemonsets for monitoring features in the future. But we can add it later.